File: src/main/java/ar/edu/unlam/tallerweb1/domain/FriendsGroupRepository.java
package ar.edu.unlam.tallerweb1.domain;

import ar.edu.unlam.tallerweb1.domain.model.FriendsGroup;
import ar.edu.unlam.tallerweb1.domain.model.User;

import java.util.List;

public interface FriendsGroupRepository {
    List<User> getMembers(FriendsGroup group);

    FriendsGroup getGroupOf(User user);
}

File: src/main/java/ar/edu/unlam/tallerweb1/domain/FriendsGroupRepository.kt
package ar.edu.unlam.tallerweb1.domain

import ar.edu.unlam.tallerweb1.domain.model.FriendsGroup
import ar.edu.unlam.tallerweb1.domain.model.User

interface FriendsGroupRepository {
    fun getMembers(group: FriendsGroup): List<User?>

    fun getGroupOf(user: User): FriendsGroup?
}
